_grains/pci_devs.py

```diff
@@ -20,7 +20,7 @@ def pci_devs():
     '''
 
     def find_devices_of_class(klass):
-        lspci = salt.utils.which('lspci')
+        lspci = salt.utils.path.which('lspci')
```

Comment:
This doesn't work with salt 2017.7.1. Either make it work with both versions (handle `AttributeError`?) or mark the PR as for R4.1 only (simply prepend "R4.1" to the title). In the latter case, conditional in `spec` isn't needed.